### PR TITLE
Update guidance on validDomains

### PR DESCRIPTION
**Pull request by erikadoyle**
Added wildcard guidance and example

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d99f9549dc548f15368b0b0662a38f5b665d1281/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 93 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e81607dc-b0d7-4eb8-a50f-4f5c73823f82) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d99f9549dc548f15368b0b0662a38f5b665d1281/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d4385a1a-c0fb-43af-b2ac-274ee5aa18f1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6959c853582ee423ae33f7852e9174980859697/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 93 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f7a2b9e8-2909-4dc9-8e4a-b99c48028809) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6959c853582ee423ae33f7852e9174980859697/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbc0678a-513c-48c1-b325-b6cf345a1cf2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/02b5b0628b10911440d34c31026d8b5a274749b1/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 93 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f40eefd7-7362-44a3-a382-3a93c7faa80a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/02b5b0628b10911440d34c31026d8b5a274749b1/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8cd9fea1-a0a6-4424-a4fa-7e1d6fcaf641) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by aeunms — APPROVED** [review]
LGTM, but would love to see Erin's feedback.

**Review by erinha — DISMISSED** [review]
These updates look good to me, thanks!

**Comment by erikadoyle** [comment]
Hi @surbhigupta12 this is ready from our side. Thanks!

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -613,7 +613,14 @@ Do **not** include the domains of identity providers you want to support in your
 Teams apps that require their own SharePoint URLs to function well, includes "{teamsitedomain}" in their valid domain list.
 
 > [!IMPORTANT]
-> Do not add domains that are outside your control, either directly or through wildcards. For example, `yourapp.onmicrosoft.com` is valid, however, `*.onmicrosoft.com` is not valid.
+> Do not add domains that are outside your control, either directly or through wildcards. For example, `*.yoursite.com` is valid, whereas `*.onmicrosoft.com` is not valid (as it isn't under your control).
+>
+> When using wildcards (`*`), the following rules apply:
```
Comment (v-ypalikila) [inline msteams-platform/resources/schema/manifest-schema.md:618]:
```suggestion
> When using wildcards, the following rules apply:
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/15b73b54e163ac18bc358014eaf3aae7eb5f4f46/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 93 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/318490d5-09c9-44e2-86c7-942b0ee6d443) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/15b73b54e163ac18bc358014eaf3aae7eb5f4f46/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ddcb3a01-f268-42c4-bea0-d0677eb5e447) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a07ba161588d45d6729dab8a1d99c832dd1ede27/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fteams-store-validation-guidelines.md) | 93 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ea7dea0b-7245-477d-b7b0-d66f52a97fbf) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a07ba161588d45d6729dab8a1d99c832dd1ede27/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b57363a6-dbd7-44cc-852d-4a7fbcc6fb86) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**